R/anovarepeatedmeasures.R

```diff
@@ -305,23 +309,25 @@ AnovaRepeatedMeasures <- function(jaspResults, dataset = NULL, options) {
     } else if (options$sumOfSquares == "type2") {
 
     tryResult <- try({
-      result <- afex::aov_car(model.formula, data=dataset, type= 2, factorize = FALSE, include_aov = TRUE)
+      result <- afex::aov_car(model.formula, data=dataset, type= 2, factorize = FALSE, 
+                              include_aov = isFALSE(options[["useMultivariateModelFollowup"]]))
```

Comment:
just for my own understanding, we do `isFALSE(options[["useMultivariateModelFollowup"]])` so that if people do *not* want multivariate follow up, we compute `aov` instead?